src/main/java/com/centralesupelec/chowchow/search/controllers/TraktSearch.java

```diff
@@ -0,0 +1,55 @@
+package com.centralesupelec.chowchow.search.controllers;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
It works well that way, I'd just like to be consistent with the way the DTOs are written (see @ShowDTO)